config/config.go

```diff
@@ -1,11 +1,34 @@
 package config
 
+import "os"
+import "path"
+
 type Config struct {
-	ListenURL string
+	ListenURL       string
+	PhotosDirectory string
 }
 
 func New() (*Config, error) {
-	return &Config{
-		ListenURL: "127.0.0.1:8000",
-	}, nil
+	c := &Config{
+		ListenURL:       "127.0.0.1:8000",
+		PhotosDirectory: "/var/hotshots",
+	}
+
+	dir, ok := os.LookupEnv("HOTSHOTS_DIR")
```

Comment:
We probably want to make `ListenURL` configurable via env var too. Another time.